### PR TITLE
chore(placement): switch to BSA

### PR DESCRIPTION
**Pull request by caugner**
## Summary

Switch to BSA for placements.

(MP-679)

### CODE REVIEW

File: cloud-function/README.md [hunk 1]
```diff
@@ -47,9 +47,8 @@ The placement handler uses the following environment variables:
 - `KEVEL_NETWORK_ID` (default: `0`) - Required for serving placements via Kevel.
 - `SIGN_SECRET` (default: `""`) - Required for serving placements.
 - `BSA_ZONE_KEYS` (default: `""`) - Required for serving placements via BSA.
-- `BSA_URL_PREFIX`(default: "https://localhost") - Where to show BSA placements
-  if enabled. Formatted like :
-  "placementname1:zonekey1;placementkey2:zonekey2...".
+- `BSA_URL_PREFIX`(default: `""`) - Where to show BSA placements if enabled.
+  Formatted like : "placementname1:zonekey1;placementkey2:zonekey2...".
```
Comment (LeoMcA) [inline cloud-function/README.md:51]: Not sure how, but these lines got mixed up in a prior commit:
```suggestion
  Formatted like : "placementname1:zonekey1;placementkey2:zonekey2...".
- `BSA_URL_PREFIX`(default: `""`) - Where to show BSA placements if enabled.
```

Comment (LeoMcA) [reply cloud-function/README.md:51]: I wonder also if the `BSA_URL_PREFIX` needs some additional explanation, like that it needs to include proto+hostname?

Comment (caugner) [reply cloud-function/README.md]: I have now removed that environment variable in 525284ae8.

File: cloud-function/src/env.ts [hunk 2]
```diff
@@ -71,8 +71,7 @@ export const SIGN_SECRET = process.env["SIGN_SECRET"] ?? "";
 export const BSA_ZONE_KEYS = Object.fromEntries(
   (process.env["BSA_ZONE_KEYS"] ?? "").split(";").map((k) => k.split(":"))
 );
-export const BSA_URL_PREFIX =
-  process.env["BSA_URL_PREFIX"] ?? "https://localhost";
+export const BSA_URL_PREFIX = process.env["BSA_URL_PREFIX"] ?? "";
```
Comment (LeoMcA) [inline cloud-function/src/env.ts:74]:
```suggestion
```

**Review by LeoMcA — APPROVED** [review]
lgtm aside from one missed `BSA_URL_PREFIX` reference
careful not to merge too soon :)

File: cloud-function/src/env.ts (hunk 2 above)
Comment (caugner) [reply cloud-function/src/env.ts]: Weird, I swear it didn't pop up in my VSCode search. 🤷

Comment (caugner) [reply cloud-function/src/env.ts]: Fixed via [ff55991](https://github.com/mdn/yari/pull/10011/commits/ff5599137fc33e9ccc140d2cac54c71a557ebc24).

**Pull request merged**